src/System.Private.CoreLib/shared/System/Resources/ResourceManager.cs

```diff
@@ -676,7 +676,7 @@ protected static CultureInfo GetNeutralResourcesLanguage(Assembly a)
         // Looks up a resource value for a particular name.  Looks in the 
```

Comment:
Nit: Is the `(CultureInfo?)` cast a few lines above this required?

Reply:
No it is not needed. I’ll clean that up.

Reply:
Fixed.